src/main/java/com/epam/test_generator/services/SuitService.java

```diff
@@ -61,4 +67,31 @@ public void removeSuit(long suitId) {
 
         caseVersionDAO.delete(suit.getCases());
     }
+
+    public void updateSuitRowNumber(List<SuitRowNumberUpdateDTO> rowNumberUpdates) {
+        if (rowNumberUpdates.isEmpty()) {
+            throw new BadRequestException("The list has not to be empty");
+        }
+
+        final Map<Long, Integer> patch = rowNumberUpdates
```

Comment:
Add unit test to check this piece of code. rowNumberUpdates for this test should contain nulls as ids and nulls as rowNumbers

Reply:
Done